content/php/concepts/math-functions/terms/acos/acos.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'floor()'
+Description: 'Returns a rounded number down to the nearest integer.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+  - 'Math Functions'
+Tags:
+  - 'Trigonometric'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+
+The built-in math function **`acos()`** determines the `arc cosine` of a given value. The inverse cosine function, which is used to identify the angle whose cosine value is equal to the specified value, is referred to as the `arc cosine` function.
+
+## Syntax
+
+```pseudo
+$result = acos(number);
+```
+
+Where `number` is the number that should be Specifies in range -1 to 1.
+
+## Example
+
+```php
+acos(0.64)

```

Comment:
The example must be full PHP code. It needs the `<?php ... ?>` tags and something line an `echo()` function to do something with the result of the function.